packages/apps/infinitepay/src/ip-list-payments.ts

```diff
@@ -0,0 +1,158 @@
+import type { AppModuleBody } from '@cloudcommerce/types';
+import type { ListPaymentsParams } from '@cloudcommerce/types/modules/list_payments:params';
+import type { ListPaymentsResponse } from '@cloudcommerce/types/modules/list_payments:response';
+import * as logger from 'firebase-functions/logger';
```

Comment:
```suggestion
import logger from 'firebase-functions/logger';
```

A não ser que você tenha visto algo que eu não vi, `import *` não é necessário aqui...

Reply:
Meu Lint tava acusando erro, confesso que não entendi 🤦‍♂️